### PR TITLE
Handles the case of 0/0 for cryptocom paired rows.

### LINKED ISSUE #4010: Crypto.com operation error
## Problem Definition

Some data in crypto.com importer causes a division error in the code. In the user case it was a amount of 0.0000003 ETH that
the CSV assigned 0 as EUR value

## Logs

```
[28/01/2022 01:22:58 GMT Standard Time] INFO rotkehlchen.api.server.pywsgi 127.0.0.1 - - "OPTIONS /api/1/import HTTP/1.1" 200 314 0.000000
[28/01/2022 01:22:58 GMT Standard Time] ERROR rotkehlchen.api.server Exception on /api/1/import [PUT]
Traceback (most recent call last):
  File "site-packages\flask\app.py", line 1516, in full_dispatch_request
  File "site-packages\flask\app.py", line 1502, in dispatch_request
  File "site-packages\flask_restful\__init__.py", line 467, in wrapper
  File "site-packages\flask\views.py", line 84, in view
  File "site-packages\flask_restful\__init__.py", line 582, in dispatch_request
  File "site-packages\webargs\core.py", line 450, in wrapper
  File "rotkehlchen\api\v1\resources.py", line 1223, in put
  File "rotkehlchen\api\rest.py", line 194, in wrapper
  File "rotkehlchen\api\rest.py", line 2083, in import_data
  File "rotkehlchen\data\importer.py", line 688, in import_cryptocom_csv
  File "rotkehlchen\data\importer.py", line 589, in _import_cryptocom_associated_entries
  File "rotkehlchen\fval.py", line 88, in __truediv__
decimal.InvalidOperation: [<class 'decimal.DivisionUndefined'>]

[28/01/2022 01:22:58 GMT Standard Time] INFO rotkehlchen.api.server.pywsgi 127.0.0.1 - - "PUT /api/1/import HTTP/1.1" 500 216 0.041305
```

### System Description
Operating system: 
Rotki version: 1.23

### PR DESCRIPTION
**Pull request by yabirgb**
Also adds luna as handled asset

Closes #4010

### CODE REVIEW

**Review by LefterisJP — COMMENTED** [review]
Why is this still a draft?

File: rotkehlchen/data/importer.py [hunk 1]
```diff
@@ -586,7 +593,11 @@ def _import_cryptocom_associated_entries(
                     base_amount_bought = deserialize_asset_amount(
                         credited_row['Amount'],
                     ) * part_of_total
-                    rate = Price(abs(quote_amount_sold / base_amount_bought))
+
+                    if quote_amount_sold != ZERO:
+                        rate = Price(abs(quote_amount_sold / base_amount_bought))
```
Comment (LefterisJP) [inline rotkehlchen/data/importer.py:598]: Hm ... isn't the problem here if `base_amount_bought` is zero? Aren't you trying to protect for division by zero?

File: docs/changelog.rst [hunk 2]
```diff
@@ -7,7 +7,9 @@ Changelog
 * :bug:`3993` Fix app bar icons misbehavior when windows is resized.
 * :bug:`3998` FTX subaccounts should now be working again.
 * :bug:`-` If binance returns a delisted market as active and rotki queries it, the entire binance trade history query will not fail.
+* :bug:`4010` Crypto.com users won't see errors for rows containing zeros.
 * :bug:`-` All Liquity events will now always be correctly queried.
+* :feature:`4010` Support for LUNA and card top ups has been added to the crypto.com importer.
```
Comment (LefterisJP) [inline docs/changelog.rst:12]: One of those two should be `-`. The other is a fix for `4010`

File: rotkehlchen/data/importer.py (hunk 1 above)
Comment (yabirgb) [reply rotkehlchen/data/importer.py:598]: Here the case where `base_amount_bought` is when `quote_amount_sold` is too but I agree that is better to protect also against division by zero by changing it

**Review by LefterisJP — APPROVED** [review]
LGTM. Thank you @yabirgb

**Pull request merged**